content/terms/resident-architecture.md

```diff
@@ -0,0 +1,55 @@
+- Immersive knowledge transfer around Sourcegraph capabilities and best practices relevant to customer teams, 
+- Strategic planning around priorities, goals, and KPIs
+- Technical partnership on leveraging Sourcegraph capabilities to solve specific technical challenges
+- Perform technical tasks on behalf of your teams such as scripting, Batch Change spec creation, Code Insight dashboard development, UI customization, organization and Own utilization, and more
+- Custom & tailored enablement for customer teams based on their tech and their needs; train the trainer
+- Joint success planning (quarterly) to ensure alignment, outcomes, and value
+- Direct technical advocacy back to internal teams on problems and feature requests
+
+
+### Direct Access
+The Resident Architect will perform the determined activities either by working collaboratively with an organization's admin or independently via direct badged-access to the customer's environment. This will be dependent on the organization’s desire and ability to grant this access to the Sourcegraph team, and the level of access that can be granted. Access is strongly recommended for this service. If your organization is interested in having Sourcegraph directly access your environment during an implementation, this should be discussed prior to purchasing the service.
+
+### Consumption Rules
+The parties agree that the following rules shall apply to the consumption of hours set forth above:
+  - (a) Availability: Unless otherwise agreed by the parties in writing, Professional Services will be performed Monday through Friday, between the hours of 8:00 a.m. – 7:00 p.m. in the Customer’s local time zone. Work outside of the hours above, work on weekends or on local holidays must be pre-approved in writing by Sourcegraph.
+  - (b) Location: Professional Services will be performed at a remote location.
+  - (c) Multiple Personnel: Sourcegrah may, at its sole discretion, choose to engage different consulting personnel for different portions of the Professional Services.
+  - (d) Any hours set forth above not consumed during the Subscription Period will expire.
+
+### Resident Architect Packages
+
+Resident Architect is priced according to both the amount of time provided to the customer and the technical level of service expected as part of the offering.
+
+- Resident Architect - Standard
+  - 260 total annual hours (approximately 5 hours per week)
+  - Primarily focused on custom onboarding and enablement work.
+- Resident Architect - Advanced
+  - 520 total annual hours (approximately 10 hours per week)
+  - Additional time beneficial for more hands on configuration work and more technical requirements from the service.
+- Resident Architect - Premium
+  - 1,040 total annual hours (approximately 20 hours per week)
+  - Good for highly technical, intensive projects often requiring hands-on custom development.
+
```

Comment:
I'd maybe add a sentence like "While we recommend distributing the hours evenly throughout the year, the total hours are allocated as an annual pool. This allows for flexibility in usage, enabling you to utilize the hours as needed to best align with your project timelines and priorities."

Reply:
Does this comment from early in the terms cover this:

"Note: hours may be able to flex monthly, subject to planning and based on project needs but should not exceed the annual allocation. Time spent towards the service will be tracked discretely and reported to Finance monthly."

Reply:
Yes!